### DISCUSSION TITLE
podroz-do-polski-z-esim-orange-flex/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#110] [General]
# podroz-do-polski-z-esim-orange-flex/

Podróżujesz do Polski i potrzebujesz nielimitowanego dostępu do internetu? Oto jak ja wykorzystałem do tego Orange Flex oraz kartę eSIM... praktycznie za darmo.

https://dariusz.wieckiewicz.org/podroz-do-polski-z-esim-orange-flex/

**Comment by xpil** [comment]
A mi się trafiła chałupa bez zasięgu 5G, 4G ani nawet 3G. Żeby odebrać SMS-a z kodem jednorazowym do banku czy sklepu, muszę iść do ogródka i udawać Statuę Wolności z ADHD, taka jego chędożona, dyszlem chrzczona mać :)

**Reply by idarek** [reply]
Ja miałem inną sytuację, którą nazwałem "czarną dziurą". Na nowo wybudowanym osiedlu, wszyscy mają pięknie pełny zasięg 5G, tylko operator zapomniał że nadajnik nie wyrobi takiego obciążenia i tak z pełnym zasięgiem jeżeli w ogóle masz internet to dobrze, a jak jest to max 500Kbps :). Pozdrawiam :)

**Reply by xpil** [reply]
Sieciowcy mają na to osobne pojęcie: "contention ratio". Jak na jednym gigabitowym łączu siedzi 1000 użytkowników i większość, za przeproszeniem, ciągnie (dane), to sobie można tego gigabita włożyć w {cenzura}.

Przy łajerlesie jest dokładnie tak samo, tylko trochę inaczej :)

**Reply by olekstomek** [reply]
Pasmo nie jest z gumy. :) W internetach stajonarnych w OLTach klienci są wpinani w porty 2,5Gbps (dla GPON) więc ta prędkość się dzieli między nich jeśli pobierają w tym samym czasie. Klientów na porcie może być różna ilość, ale może to być 64, 128 lub więcej, "to zależy". Ale często ruch "mija się" - nie wszyscy pobierają w tym samym czasie. Łaczę rzadko kiedy jest wysycane na 100% przez długi czas (chyba jedynie przy pobieraniu bo przykładowo YouTube to skoki tranferu i pobieranie do bufora partiami). I nie bez powodu w regulaminach i umowach u wielu/wszystkich ISP jest napisana informacja o prędkości maksymalnej, zwykle dostępnej i minimalnej, i że dana prędkość jest gwarantowana przez np. 80% doby. Więc tutaj jest to zabezpieczenie i wieczorem transfery rzeczywiście mogą spadać, a szczególnie jak się trafią na portach power userzy.

**Comment by spexpl** [comment]
A ja nie do końca rozumiem te problemu z tethering. Słyszałem o tym problemie w materiałach o internecie mobilnym w USA, teraz czytam w UK. A ja jakoś w PL nie miałem z tym nigdy problemu.

**Comment by olekstomek** [comment]
Trochę od strony technicznej kilka stron PDFa jak działa Orange Flex: https://www.tmforum.org/wp-content/uploads/2021/02/TM-Forum-Excellence-Awards-Case-Study-CX_Orange-Flex.pdf (nie jest to najświeższy dokument ale zawsze daje jakiś pogląd na rozwiązania i architekturę). :)
